.github/workflows/publishpackage.yml

```diff
@@ -1,12 +1,26 @@
 name: Publish Python Package
-
-on: 
+on:
   push:
     branches:
       - master
-
 jobs:
+  test:
+    runs-on: ubuntu-18.04
+    name: Run Tests/Linters
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install
+        run: |
+          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3
+          source $HOME/.poetry/env
```

Comment:
Why not just `pip install poetry`?

Reply:
That's what I typically do, but it seems the poetry maintainers [recommend against it](https://python-poetry.org/docs/#installing-with-pip). Incidentally, I had some issues with `pip install poetry` when playing with this, but if you strongly prefer that approach, I can give it another go.